teamcity/tox.ini

```diff
@@ -3,8 +3,9 @@ isolated_build = true
 minversion = 2.0
 basepython = py38
 envlist =
-    py{27,38}
-
+    py27-legacy
+    py38-openmetrics
+    py38
```

Comment:
Should we have "legacy" in the name here too?

Reply:
I named the py27 environment "legacy" because the project "autodiscovery" feature is not supported in py27. The `py27-legacy` environment uses the original legacy instance config option, `build_configuration` instead of `projects`.

Reply:
IC.